### ISSUE TITLE
Pytest discovery error: python.envFile not working for it

### ISSUE THREAD

**Issue by alanwilter** [microsoft/vscode#136750]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.62.0
- OS Version: Darwin x64 20.6.0 (macOS Big Sur 11.6.1)

I use pytest and my project depends on an env var defined in a `.env` file as `VCF_FILE=schema/small_demo.vcf.gz`

If I do:
```bash
export VCF_FILE=schema/small_demo.vcf.gz
code .
```
VScode will populate tab `Testing`.

However if I start VSCode via mouse click on the app, `python.envFile` does not get `VCF_FILE` and then `Testing` will fail with:
```python
[ERROR 2021-10-9 13:5:0.961]: Error discovering pytest tests:
 [r [Error]: ImportError while loading conftest '/Users/alan/Programmes/phenopolis_browser/tests/conftest.py'.
tests/conftest.py:4: in <module>
    from views import application, APP_ENV, VERSION
views/__init__.py:36: in <module>
    variant_file = VCF(os.getenv("VCF_FILE"))
cyvcf2/cyvcf2.pyx:257: in cyvcf2.cyvcf2.VCF.__init__
    ???
cyvcf2/cyvcf2.pyx:186: in cyvcf2.cyvcf2.HTSFile._open_htsfile
    ???
E   OSError: Cannot open '<class 'NoneType'>' for writing.
```

Bottom line, VSCode `python.envFile` is not working for python environment and hence  discovering pytest tests will fail.
